### PR TITLE
img.shields.io has ERR_CONNECTION_RESET

### PR DESCRIPTION
**Pull request by sakharovsergey**

### CODE REVIEW

**Comment by tlvince** [comment]
Seems fine here?

```
❯ curl -I https://img.shields.io/badge/build-passing-brightgreen.svg
HTTP/2 200
date: Thu, 16 May 2019 16:23:46 GMT
content-type: image/svg+xml;charset=utf-8
set-cookie: __cfduid=d2618b678bcc99ff86bb2e49443b6d0721558023826; expires=Fri, 15-May-20 16:23:46 GMT; path=/; domain=.shields.io; HttpOnly
cache-control: max-age=86400
last-modified: Mon, 13 May 2019 12:16:27 GMT
cf-cache-status: HIT
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
server: cloudflare
cf-ray: 4d7eb2b10eb83474-LHR
```

**Comment by sakharovsergey** [comment]
```
> $ curl -vI https://img.shields.io/badge/build-passing-brightgreen.svg                                                                                                  [±master ●]
*   Trying 104.28.20.21...
* TCP_NODELAY set
* Connected to img.shields.io (104.28.20.21) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to img.shields.io:443 
* stopped the pause stream!
* Closing connection 0
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to img.shields.io:443 
```

**Comment by sakharovsergey** [comment]
Снимок экрана 2019-05-16 в 21 32 04
maybe only for Russia region 🤷🏼‍♂️

File: cli.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ const getBadge = report => {
   const coverage = report.total.statements.pct
   const colour = getColour(coverage)
 
-  return `https://img.shields.io/badge/Coverage-${coverage}${encodeURI('%')}-${colour}.svg`
+  return `https://badgen.net/badge/coverage/${coverage}${encodeURI('%')}/${colour}`;
+  // return `https://img.shields.io/badge/Coverage-${coverage}${encodeURI('%')}-${colour}.svg`
```
Comment (tlvince) [inline cli.js:25]: Could you delete this line, please?

**Review by tlvince — CHANGES_REQUESTED** [review]
Some info on [Badgen vs shields.io](https://www.reddit.com/r/javascript/comments/91m0cn/badgen_fast_badge_generating_service_shieldsio/).

Am happy for this to go in, but could you fix the [lint warning](https://travis-ci.org/tlvince/make-coverage-badge/builds/533410793#L478) and reformat the commit message into [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) form? Thanks!

**Comment by tlvince** [comment]
Could you also update the shields.io link and badges used in the readme?

**Pull request closed without merging**